docs/source/membership/membership.md

```diff
@@ -1,43 +1,86 @@
-# Membership
+# Membership Service Provider (MSP)
 
-If you've read through the documentation on [identity](../identity/identity.html)
-you've seen how a PKI can provide verifiable identities through a chain
-of trust. Now let's see how these identities can be used to represent the
-trusted members of a blockchain network.
+## Why do I need an MSP?
 
-This is where a **Membership Service** Provider (MSP) comes into play ---
-**it identifies which Root CAs and Intermediate CAs are trusted to define
-the members of a trust domain, e.g., an organization**, either by listing the
+Because Fabric is a permissioned network, blockchain participants need a way to
+prove their identity to the rest of the network in order to transact on the
+network. If you've read through the documentation on [identity](../identity/identity.html)
+you've seen how a PKI can provide verifiable identities through a chain
+of trust. How is that chain of trust used by the blockahin network?
+Recall in the credit card scenario that the Certificate Authority is
+like a card provider — it dispenses many different types of verifiable
+identities. An MSP, on the other hand, is like the list of card providers
+accepted by the store, determining which identities are the trusted
+members (actors) of the store  payment network.
+
+![MSP1a](./membership.msp.diagram.png)
+
+*Identities are similar to your credit cards that are used to prove
+you can pay. The MSP is similar to the list of accepted credit cards.*
+
+Or consider a consortium of banks that each operate peer
+nodes that both endorse and validate transactions. They could also optionally
+operate ordering nodes. The Bank A account holders have identities affiliated
+with Bank A, and the Bank B account holders have identities affiliated
+with Bank B. But the account holders don't operate nodes, they simply
+interact with the system from their mobile or web application.
+
+So how does the network recognize and differentiate the identities?
```

Comment:
I would switch the two senences to match the order in the introduction and order of importance:

So how does the network recognize and differentiate the identities? MSP's are used to define the organizations that belong to the network, whose members are trusted by other network members. MSPs are also the mechanism that provide members with a set of roles and permissions within the network.

Reply:
Done.